bigtable/docs/snippets.py

```diff
@@ -425,5 +425,101 @@ def test_bigtable_set_iam_policy_then_get_iam_policy():
     assert len(policy.bigtable_admins) > 0
 
 
+def test_bigtable_admins_policy():
+    # [START bigtable_admins_policy]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable.policy import Policy
+    from google.cloud.bigtable.policy import BIGTABLE_ADMIN_ROLE
+
+    # [END bigtable_admins_policy]
+
+    service_account_email = Config.CLIENT._credentials.service_account_email
+
+    # [START bigtable_admins_policy]
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    instance.reload()
+    new_policy = Policy()
+    new_policy[BIGTABLE_ADMIN_ROLE] = [Policy.service_account(service_account_email)]
+
+    policy_latest = instance.set_iam_policy(new_policy)
+    policy = policy_latest.bigtable_admins
+    # [END bigtable_admins_policy]
+
+    assert len(policy) > 0
+
+
+def test_bigtable_readers_policy():
+    # [START bigtable_readers_policy]
```

Comment:
There are repeated tags here. I don't know how the parser is going to handle that. @jmdobry , is this a pattern that works?

Reply:
This has been already used and works fine.